pkg/config/validation.go

```diff
@@ -358,10 +358,6 @@ func validateManagedJobsNamespaceSelector(c *configapi.Configuration) field.Erro
 		return allErrs
```

Comment:
As discussed in the other thread, choose either:
- drop all uses of the feature gate ManagedJobsNamespaceSelector, and drop tests
- keep uses of ManagedJobsNamespaceSelector, but then also keep tests

Because keeping code, but dropping tests feels inconsistent.